Hydrator/HydrateWithMapTrait.php

```diff
@@ -28,7 +28,7 @@ protected function hydrateUserWithAttributesMap(
 
             $ldapValue = $ldapUserAttributes[$attr['ldap_attr']];
 
-            if (!array_key_exists('count', $ldapValue) || $ldapValue['count'] == 1) {
+            if ((!array_key_exists('count', $ldapValue) && count($ldapValue) == 1) || (array_key_exists('count', $ldapValue) && $ldapValue['count'] == 1)) {
```

Comment:
This could be simplified as:

 `(array_key_exists('count', $ldapValue) && $ldapValue['count'] == 1) || count($ldapValue) === 1`